src/libs/actions/Report.ts

```diff
@@ -5043,6 +5079,254 @@ function moveIOUReportToPolicy(reportID: string, policyID: string) {
     API.write(WRITE_COMMANDS.MOVE_IOU_REPORT_TO_EXISTING_POLICY, parameters, {optimisticData, successData, failureData});
 }
 
+/**
+ * Moves an IOU report to a policy by converting it to an expense report
+ * @param reportID - The ID of the IOU report to move
+ * @param policyID - The ID of the policy to move the report to
+ */
+function moveIOUReportToPolicyAndInviteSubmitter(reportID: string, policyID: string) {
+    const iouReport = allReports?.[`${ONYXKEYS.COLLECTION.REPORT}${reportID}`];
+    const policy = getPolicy(policyID);
+    const isPolicyAdmin = isPolicyAdminPolicyUtils(policy);
+
+    // This flow only works for IOU reports
+    if (!policy || !iouReport || !isPolicyAdmin || !isIOUReportUsingReport(iouReport)) {
+        return;
+    }
+
+    // We do not want to create negative amount expenses
+    if (ReportActionsUtils.hasRequestFromCurrentAccount(reportID, iouReport.managerID ?? CONST.DEFAULT_NUMBER_ID)) {
+        return;
+    }
+
+    // Generate new variables for the policy
+    const policyName = policy.name ?? '';
+    const iouReportID = iouReport.reportID;
+    const ownerAccountID = iouReport.ownerAccountID;
+    const expenseChatReportID = getPolicyExpenseChat(ownerAccountID, policyID)?.reportID;
+
+    if (!expenseChatReportID) {
+        return;
+    }
```

Comment:
We need to remove this check. The submitter is not a policy member so they won't have a policyExpenseChat. We should instead, optimistically generate it and pass the optimistic `policyExpenseChatReportID` and `policyExpenseCreatedReportActionID` (this is the CREATED action we add to the report) to the API